```
builder for '/nix/store/sihd6b4mnibpfr1rcjjp25plhshvw75d-ffmpeg-full-5.1.drv' failed with exit code 2; last 10 log lines:
  CC  tests/api/api-threadmessage-test.o
  STRIP      tests/checkasm/x86/checkasm.o
  skipping strip -x tests/checkasm/x86/checkasm.o
  CC    tests/api/api-flac-test.o
  CC  tests/api/api-seek-test.o
  HOSTCC      tests/audiogen.o
  HOSTCC       tests/videogen.o
  fate-imf-cpl-with-repeat requires external samples and SAMPLES not specified
  make: *** [tests/Makefile:285: fate-imf-cpl-with-repeat] Error 1
  make: *** Waiting for unfinished jobs....
```

--- a/pkgs/development/libraries/ffmpeg-full/default.nix
+++ b/pkgs/development/libraries/ffmpeg-full/default.nix
@@ -258,6 +258,10 @@ stdenv.mkDerivation rec {
       --replace /usr/local/lib/frei0r-1 ${frei0r}/lib/frei0r-1
     substituteInPlace doc/filters.texi \
       --replace /usr/local/lib/frei0r-1 ${frei0r}/lib/frei0r-1
+  '' +
+  # ffmpeg 5.1 https://trac.ffmpeg.org/ticket/9841
+  ''
+    substituteInPlace tests/Makefile --replace 'include $(SRC_PATH)/tests/fate/imf.mak' ""
   '';
 
   configurePlatforms = [];

